docs/dml-of-carbondata.md

```diff
@@ -454,6 +461,10 @@ CarbonData DML statements are documented here,which includes:
 
```

Comment:
Can specify without providing expression, it will delete all records from table

Reply:
Done